lib/ex_unit/lib/ex_unit/formatter.ex

```diff
@@ -110,7 +110,10 @@ defmodule ExUnit.Formatter do
         ""
       end
 
-    "Finished in #{format_us(total_us)} seconds " <>
+    :test_helper.maybe_write_coverdata()
```

Comment:
this is the easiest way i could find of generating the coverage report after running all the tests